[DOC] update scalars documentation & split API methods description to its own page (#796)
yeatmanlab/pyAFQ: docs/source/usage/scalars.rst

> The pyAFQ Custom Structural Properties API

Reviewer: ```suggestion
The pyAFQ Custom Tissue Properties API
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> pyAFQ has a system for users to add additional structural properties

Reviewer: ```suggestion
Per default, pyAFQ calculates tract profiles of the FA and MD in every bundle that it recognizes. 
However, there is also a system for users to add tissue properties
```

> or other additional images (beyond the several we calculate by default).

Reviewer: ```suggestion
or other scalar images (i.e., 3D images with one numeric value per voxel).
```
